src/adagrams.js

```diff
@@ -1,15 +1,133 @@
+
+
+
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
-};
+  
+  const letterPool = {
```

Comment:
This can be declared outside of the function so that it doesn't have to be recreated each time the function is called.